src/schemas/40-10007/schema.js

```diff
@@ -124,10 +124,14 @@ const emailFormat = {
   format: 'email'
 };
 
-definitions.address.required = ['street'];
+definitions.address.required = ['street', 'city', 'state', 'postalCode'];
 definitions.address.properties.street.maxLength = 20;
 definitions.address.properties.street2.maxLength = 20;
 definitions.address.properties.city.maxLength = 20;
+definitions.address.oneOf.forEach((obj) => {
+  obj.properties.postalCode.maxLength = 5;
+  obj.properties.state.maxLength = 3;
```

Comment:
The Mexican states are longer than three letters. Does this schema need to adjust those?

Reply:
Maybe.  We currently don't show a `state` field on the frontend when Mexico is selected.
If we go with a solution of submitting a placeholder/default `state` value for all non-USA/Canada countries, then it probably does not need to change.

If we actually add the state field to the form for Mexico then it will need to change.

I did create a ticket for this issue here:  https://github.com/department-of-veterans-affairs/vets.gov-team/issues/13461

I have emailed EOAS stakeholders for their feedback.

Reply:
Ah, got it. I was confused about why the Mexican states are even listed in the schema in that case. But that's just the result of us using the common definition. 👍